CHANGELOG

```diff
@@ -1,3 +1,24 @@
+18 December 2016: mitmproxy 1.0
+
+    * Mitmproxy is now Python 3 only
```

Comment:
s/Mitmproxy/All mitmproxy tools/
This should include `mitmproxy`, `mitmdump`, `mitmweb`, `pathod`, and `pathoc`.